script.js

```diff
@@ -0,0 +1,95 @@
+let defaultResult = 0;
+
+let currentResult = defaultResult;
+
+let logEntries = [];
```

Comment:
can this be a const ?

Reply:
I think I have reassigned a value this variable , so it must be let.

Reply:
If you have learnt about primitive type vs reference type then you should replace it with const.
You haven't then dont worry. You will understand this comment of mine after some days. Plz take a note of this !

Reply:
ok. Will see this in some days.

Reply:
Hi Sir @faran4engg , I have studied primitive type vs reference type, and replaced let with const.